subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/execution/ExecuteActionsTaskExecuterTest.groovy

```diff
@@ -103,7 +108,7 @@ class ExecuteActionsTaskExecuterTest extends Specification {
     }
     def state = new TaskStateInternal()
     def taskProperties = Stub(TaskProperties) {
-        getInputPropertyValues() >> { { -> ImmutableSortedMap.of() } as Supplier }
+        getInputProperties() >> ImmutableSortedSet.of()
```

Comment:
IIUC, the removed call was never actually invoked, right? Do we now need the new call?